components/Modal/CreateImage.js

```diff
@@ -147,6 +152,7 @@ class CreateImage extends React.Component {
                   <label
                     className="col-sm-3 control-label"
                     htmlFor="textInput2-modal-markup"
+                    data-id="cmpsr-modal-markup2"
```

Comment:
Instead of adding a `data-id` attribute here, could you add an `id` to the `<select>` as noted in a [previous comment](https://github.com/weldr/welder-web/pull/375/files#diff-886584fb195b433a216cecc7e230cd8cR27)